### ISSUE TITLE
monster spawns mismatch

### ISSUE THREAD

**Issue by stingysaffa** [PaimaStudios/tarochi-game#116]
![R](https://github.com/PaimaStudios/tarochi-game/assets/163665155/675de7ee-513a-4b20-a702-5f332133ad08)
See image - monster spawning thats not on the 4 displayed at the bottom of the screen.

**Comment by acedward** [comment]
Did this happen more than once? e.g., Did you fight more than one Dripplingo?

**Comment by stingysaffa** [comment]
> Did this happen more than once? e.g., Did you fight more than one Dripplingo?

yes! happens all the time . different areas too - different monster. Which ever monster the rare - epic - legen replaces , is the one that spawns.

**Comment by stingysaffa** [comment]
> Did this happen more than once? e.g., Did you fight more than one Dripplingo?

![Uploading hh.JPG…]()

**Comment by stingysaffa** [comment]
![hh](https://github.com/PaimaStudios/tarochi-game/assets/163665155/c29e4915-940d-47b6-a216-bf7fbc1bec9f)

**Comment by stingysaffa** [comment]
Any fix in the works? trying to grind but this is making it almost impossible :(
